vowpalwabbit/global_data.h

```diff
@@ -511,15 +510,15 @@ struct vw
   std::stack<VW::LEARNER::base_learner* (*)(VW::config::options_i&, vw&)> reduction_stack;
 
   // Prediction output
-  v_array<int> final_prediction_sink;  // set to send global predictions to.
-  int raw_prediction;                  // file descriptors for text output.
+  v_array<VW::io::writer*> final_prediction_sink;  // set to send global predictions to.

```

Comment:
Since this owns the objects, can we just make this `vector<unique_ptr<writer>>`?